google-http-client/src/main/java/com/google/api/client/http/HttpRequest.java

```diff
@@ -695,6 +698,25 @@ public HttpRequest setFollowRedirects(boolean followRedirects) {
     return this;
   }
 
+  /**
+   * Returns whether to encode redirect urls.
+   *
+   */
+  public boolean getEncodeRedirectUrls() {
+    return encodeRedirectUrls;
+  }
+
+  /**
+   * Sets whether to encode redirect urls.
+   *
+   * <p>The default value is {@code true}.
+   *
+   */
+  public HttpRequest setEncodeRedirectUrls(boolean encodeRedirectUrls) {
+    this.encodeRedirectUrls = followRedirects;
```

Comment:
`followRedirects` seems wrong.

And this isn't only about encoding. Decoding is disabled for redirection too. `setUseRawRedirectUrls()`?